### PR TITLE
Fix 3D Convexity measurement.

### PR DESCRIPTION
**Pull request by tibuch**
Fix for this issue https://github.com/imagej/imagej-ops/issues/483.

New:
3D convexity = volume/volumeConvexHull

Old:
3D convexity = surfaceArea(ConvexHull)/surfaceArea

### CODE REVIEW

File: src/test/java/net/imagej/ops/geom/MeshFeatureTests.java [hunk 1]
```diff
@@ -98,7 +101,7 @@ public void convexHull3D() {
 	@Test
 	public void convexityMesh() {
 		// formula verified and ground truth computed with matlab
-		assertEquals(Ops.Geometric.Convexity.NAME, 0.983930494866521,
+		assertEquals(Ops.Geometric.Convexity.NAME, 0.84564860426,
```
Comment (imagejan) [inline src/test/java/net/imagej/ops/geom/MeshFeatureTests.java]: This is now exactly the same as ` Ops.Geometric.Solidity`.

**Comment by tibuch** [comment]
Since the new _Convexity_ is equal to _Solidity_ this seems not to be the way to go.

As far as I can tell we have three options:
1. Keep _Convexity_ as it is. (Not mapping to [0, 1])
2. _Convexity_ == _Solidity_ (This PR)
3. Remove _Convexity_.

**Comment by stelfrich** [comment]
IMHO, changing the 3D convexity implementation to emulate the 3D solidity implementation isn't a viable alternative since the transfer from 2D to 3D convexity seems obvious (boundary to surface). Isn't it a completely different story (interpretation in contrast to implementation), that this doesn't map to `[0, 1]`?

**Comment by tibuch** [comment]
I agree with @stelfrich. What is your opinion @imagejan?

**Comment by imagejan** [comment]
I agree, my comment above intended to say: it's wrong to change this calculation to something that's not backed by the theoretical definition.
But then: isn't the convexity still supposed to be between 0 and 1, from the mathematical term? Anything else is hard to interpret, IMHO.

I guess it all comes down to: I'd like to see a correct mathematical _definition_ of 3D Convexity, and this (as well as its correct _interpretation_) should be clearly documented - in the javadoc, and (ideally automatically) as well in downstream applications, e.g. in the node descriptions of the Feature Calculator in KNIME.

**Comment by ctrueden** [comment]
@tibuch Was a consensus reached on what to do about this?

**Comment by imagejan** [comment]
I think we can stick with option 1:

> 1. Keep Convexity as it is. (Not mapping to [0, 1])

and close this PR.

As mentioned in #483, I just found it counter-intuitive that convexity values can be higher than 1, so I still think this should be well documented somewhere.

**Comment by ctrueden** [comment]
Thanks, everyone.

**Pull request closed without merging**